lib/encrypt_io.dart

```diff
@@ -9,3 +9,10 @@ Future<T> parseKeyFromFile<T extends RSAAsymmetricKey>(String filename) async {
   final parser = RSAKeyParser();
   return parser.parse(key) as T;
 }
+
+T parseKeyFromFileSync<T extends RSAAsymmetricKey>(String filename) {
+  final file = File(filename);
+  final key = await file.readAsStringSync();
```

Comment:
why there is `await` here? it's not `async`

Reply:
bad copy pasta on my part, I think..

Reply:
#249 fixes this